templates/postcss.config.js

```diff
@@ -1,23 +1,7 @@
-const purgecss = require("@fullhuman/postcss-purgecss")({
-  content: [
-    "./output/**/*.html",
-    "./frontend/**/*.js",
-    "./src/**/*.md",
-    "./src/**/*.html",
-    "./src/**/*.liquid",
-  ],
-  defaultExtractor: (content) => {
-    const broadMatches = content.match(/[^<>"'`\s]*[^<>"'`\s:]/g) || [];
-    const innerMatches = content.match(/[^<>"'`\s.()]*[^<>"'`\s.():]/g) || [];
-    return broadMatches.concat(innerMatches);
-  },
-});
-
 module.exports = {
   plugins: [
-    require("postcss-import"),
-    require("tailwindcss"),
+    require("tailwindcss")("tailwind.config.js"),
     require("autoprefixer"),
-    ...(process.env.NODE_ENV === "production" ? [purgecss] : []),
-  ],
+    require("postcss-import")
```

Comment:
I believe this should be the first plugin.

https://tailwindcss.com/docs/using-with-preprocessors#build-time-imports